discovery-consul/src/test/resources/reference.conf

```diff
@@ -0,0 +1,3 @@
+akka {
+
+}
```

Comment:
trailing \n is needed in reference confs to avoid issues in general, but here perhaps we can just remove it

Reply:
(the file I mean)

Reply:
removed